src/helpers/feature.js

```diff
@@ -38,6 +39,17 @@ export function requestAnimationFrame(callback) {
   return _requestAnimationFrame.call(window, callback);
 }
 
+export function isClassListSupported() {
+  return !!document.documentElement.classList;
+}
+
+export function isTextContentSupported() {
+  return document.createTextNode('test').textContent;
```

Comment:
```suggestion
  return !!document.createTextNode('test').textContent;
```
Missing to Boolean conversion.

Reply:
You're right! Thanks @budnix :+1: